src/backend/utils.js

```diff
@@ -0,0 +1,6 @@
+export function searchComponent (el) {
```

Comment:
I think we could have a more explicit name like `findRootComponent`

Reply:
We don't find the root component, we find the closest component that contains the element.

Reply:
yes, sorry, I updated the comment right away 😆